kernel/src/hil/spi.rs

```diff
@@ -121,7 +121,9 @@ pub trait SpiMaster {
 /// SPIMasterDevice provides a chip-specific interface to the SPI Master
 /// hardware. The interface wraps the chip select line so that chip drivers
 /// cannot communicate with different SPI devices.
-pub trait SPIMasterDevice {
+pub trait SpiMasterDevice {
+    type ChipSelect: Copy;
```

Comment:
So, now the only difference between `SpiMasterDevice` and `SpiMaster` is that `SpiMasterDevice` doesn't have `hold_low` and `release_low`.